### ISSUE TITLE
I have issue running project

### ISSUE THREAD

**Issue by tinkerbaj** [leomorpho/goship#1]
When I try to run make seed I get this error
```
make seed
go run cmd/seed/main.go
(*ent.User)(<nil>)
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x68 pc=0x18e10ec]

goroutine 1 [running]:
github.com/mikestefanello/pagoda/seeder.SeedUsers.func2(0x0, {0x0?, 0x0?}, {0x0?, 0x80?, 0x3316920?}, {0x21dcd30, 0x2}, {0x1defd71, 0xc})
        /home/bajro/GoShip-main/seeder/seeder.go:103 +0xec
github.com/mikestefanello/pagoda/seeder.SeedUsers(0xc00101fb08, 0xc00126dba0, 0x1)
        /home/bajro/GoShip-main/seeder/seeder.go:135 +0x4c9
main.main()
        /home/bajro/GoShip-main/cmd/seed/main.go:12 +0x2d
exit status 2
make: *** [Makefile:149: seed] Error 1
```

**Comment by leomorpho** [comment]
Hey thanks for the comment! I tried reproducing it (with a fresh `git clone` in `/tmp` and fresh containers), but couldn't reproduce it. Not quite sure what's going on here. I will keep an eye out for it.

Let me know if there's more context I might have missed in trying to reproduce this, thanks!!

**Comment by leomorpho** [comment]
Hey thanks for your message and trying out GoShip!
I can reproduce your error when I run the seeder more than once on the same containers. While I never made it idempotent before, I added a rough check for idempotency and it can now be run multiple times in a row without issues.
Sorry for the unfriendly crash.
I also updated the Readme to have only 1 command to set up a fully working instance. 
Let me know if you have any more questions!
